ios/RNMobileDeviceManager.h

```diff
@@ -10,5 +10,6 @@
 #import <UIKit/UIKit.h>
 
 @interface MobileDeviceManager : NSObject <RCTBridgeModule, ManagedAppConfigSettingsDelegate>
+@property dispatch_semaphore_t asamSem;
```

Comment:
This should not be considered a public member (define in the implementation file, not header)